sources/europe/pl/PoznaOrthophotomap2016(aerialimage).geojson

```diff
@@ -1,25 +1,31 @@
 {
-    "type": "Feature",
-    "properties": {
-        "id": "poznan-ortofotomapa2016",
-        "name": "Poznań: Orthophotomap 2016 (aerial image)",
-        "type": "wms",
-        "url": "http://wms1.geopoz.poznan.pl:6080/arcgis/services/sip/ortofotomapy/MapServer/WmsServer?FORMAT=image/jpeg&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=ortofotomapa_2016_image&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
-        "start_date": "2016",
-        "end_date": "2016",
-        "country_code": "PL",
-        "available_projections": [
-            "EPSG:4326",
-            "EPSG:2177",
-            "EPSG:2180",
-            "EPSG:3857"
-        ],
-        "attribution": {
-            "text": "Zarząd Geodezji i Katastru Miejskiego GEOPOZ"
-        },
-        "max_zoom": 23,
-        "category": "historicphoto"
+  "type": "Feature",
+  "properties": {
+    "id": "poznan-ortofotomapa2016",
+    "name": "Poznań: Orthophotomap 2016 (aerial image)",
+    "type": "wms",
+    "url": "http://wms1.geopoz.poznan.pl/arcgis/services/sip/ortofotomapy/MapServer/WMSServer?LAYERS=ortofotomapa_2016_image&STYLES=default&FORMAT=image/jpeg&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+    "license_url": "https://www.geopoz.pl/bip/index.php?t=200&id=2449",
```

Comment:
This link just seems to take me to the home page? Is there somewhere that lists the licensing?